lib/services/elasticsearch.js

```diff
@@ -788,7 +797,9 @@ class ElasticSearch extends Service {
   getMapping(request) {
     const esRequest = initESRequest(request);
 
-    return this.esWrapper.getMapping(esRequest);
+    const includeKuzzleInfo = Boolean(request.input.args.includeKuzzleInfo) || false;
```

Comment:
`Boolean()` cannot return anything else than a boolean value.
For instance, `Boolean(undefined)` results in `false`.

```suggestion
    const includeKuzzleInfo = Boolean(request.input.args.includeKuzzleInfo);
```